### PR TITLE
Break props, data and computed in different sections and make them searchable 

### LINKED ISSUE #258: State Inspector: break props/data/computed into separate sections
Open to PRs!

### PR DESCRIPTION
**Pull request by markussorg**
Fixes #258. Needs: npm install (added lodash.groupBy as dependency)

devtools

### CODE REVIEW

**Comment by andreiglingeanu** [comment]
Nice looking!

**Comment by yyx990803** [comment]
Nice work! How does it look when it's narrow? Does it go vertical? :)

File: src/devtools/views/components/ComponentInspector.vue [hunk 1]
```diff
@@ -84,4 +110,20 @@ export default {
...
   margin 0 10px
+
+.data
+  padding: 20px 0px
+
+.data-container
+  float: left
+  min-width: 33%
+  padding: 0px 10px
+
```
Comment (crswll) [inline src/devtools/views/components/ComponentInspector.vue]: I'd move the `float` and `min-width` for `.data-container` into a media query here so it stacks when we don't have the horizontal space. I'd *try* this. I didn't test it at all but should get you close.

```stylus
@media (min-width: $wide)
  float left
  width 33.333%
  overflow overlay
  ```

**Comment by markussorg** [comment]
Added responsive styling. @crswll: Took a slightly different apprach with flexbox. 

responsive

**Comment by markussorg** [comment]
Added data search. Fixes #252

data_search

**Pull request merged**